src/components/playlist-item/index.tsx

```diff
@@ -93,6 +107,24 @@ export const PlaylistItem = withText(translates)(({item, active, onSelect, plugi
         </div>
       );
     }
+    if (type === MediaTypes.Document) {
```

Comment:
should this be in common ?

Reply:
that requires refactoring of other playlist-item type icons, for now playlist icons keeps in local repo